homeassistant/components/vizio/config_flow.py

```diff
@@ -68,7 +68,11 @@ def _get_config_schema(input_dict: dict[str, Any] = None) -> vol.Schema:
             vol.Required(
                 CONF_DEVICE_CLASS,
                 default=input_dict.get(CONF_DEVICE_CLASS, DEFAULT_DEVICE_CLASS),
-            ): vol.All(str, vol.Lower, vol.In([DEVICE_CLASS_TV, DEVICE_CLASS_SPEAKER])),
+            ): vol.All(
+                str,
+                vol.Lower,
+                vol.Coerce(MediaPlayerDeviceClass),
```

Comment:
This is a change of functionnality: it allows `receiver`

Reply:
@raman325 was this voluntary? I missed the earlier conversation when I posted this.

Reply:
no that's a good point, I missed that receiver is also a device class. @tkdrob can we revert this commit?